### PR TITLE
Add Less

### PR DESCRIPTION
**Pull request by LucasBassetti**
[https://github.com/LucasBassetti/awesome-less](https://github.com/LucasBassetti/awesome-less)

Less is a CSS pre-processor, meaning that it extends the CSS language, adding features that allow variables, mixins, functions and many other techniques that allow you to make CSS that is more maintainable, themeable and extendable. It would be a good idea to have a list of awesome Less frameworks, libraries, style guides, articles, and resources for people to discover more awesome things.

# By submitting this pull request I confirm I've read and complied with the below requirements.

Failure to properly do so will just result in the pull request being closed and everyone's time wasted. Please read it twice. Most people miss many things.

- [x] I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- [x] This pull request has a descriptive title. For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- [x] The entry in the Awesome list should include a short description about the list project/theme (not the list itself), and should be added at the bottom of the appropriate category.
- [x] The list I added
	- [x] **has been around for at least 30 days,**
	- [x] is the result of hard work and is the best I could possibly produce,
	- [x] is a non-generated Markdown file in a GitHub repo,
	- [x] is not a duplicate,
	- [x] includes a succinct description of the project/theme at the top ([example](https://github.com/willempienaar/awesome-quantified-self)),
	- [x] includes a project logo/illustration (whenever possible) at the top right of the readme ([example](https://github.com/sindresorhus/awesome-electron)). (The logo should link to the project website or any relevant website. The logo should be high-DPI, so set it to maximum half the width of the original image.),
	- [x] only has awesome items **with descriptions** *(Awesome lists are curations of the best, not of everything)*,
	- [x] contains the [awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge) on the right side of the list heading,
	- [x] has a Table of Contents section **named `Contents`** as the first section,
	- [x] has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license) *(Meaning something like CC0, NOT a code licence like MIT, BSD, Apache, etc)* *(If you use a license badge, it should be SVG, not PNG)*,
	- [x] has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines),
	- [x] doesn't include a Travis badge,
	- [x] has been checked for inconsistent formatting, spelling, and grammar:
		- [x] Each link description starts with an uppercase character and ends with a period.
		- [x] Drop all the `A` / `An` prefixes in the descriptions.
		- [x] Consistent naming. For example, `Node.js`, not `NodeJS` or `node.js`.

### CODE REVIEW

**Comment by davisonio** [comment]
Please read requirements

**Comment by LucasBassetti** [comment]
updated

File: readme.md [hunk 1]
```diff
@@ -190,6 +190,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [webpack](https://github.com/d3viant0ne/awesome-webpack) - Module bundler.
 - [Browserify](https://github.com/ungoldman/awesome-browserify) - Module bundler.
 - [Sass](https://github.com/Famolus/awesome-sass) - CSS preprocessor.
+- [Less](https://github.com/LucasBassetti/awesome-less) - Less preprocessor.
```
Comment (sindresorhus) [inline readme.md]: It's a `CSS preprocessor.`

**Comment by sindresorhus** [comment]
> includes a project logo/illustration (whenever possible) at the top right of the readme (example). (The logo should link to the project website or any relevant website. The logo should be high-DPI, so set it to maximum half the width of the original image.),

This should be the actual Less logo.

> Consistent naming. For example, Node.js, not NodeJS or node.js.

Not followed. You both Less, LESS, Less.js, LessJS. Should be Less.

---

- The `About` section is too long. Make it more focused on what Less is and less history. I don't think it should be a separate section, but rather placed below the list title. Example: https://github.com/woop/awesome-quantified-self
- `Official Less Guide - Official Less guide.` - Don't add description when it's identical to the title. That's moot.
- `Tutorialzone - Learn Less in 10 Minutes (or Less).` - Here you have the title and description reversed.
- Drop the `Other Awesome Lists` section.
- `for Less.`, `to Less`, etc, are moot. This is a list for Less. Things being about Less is implied. Same with titles, like `Less Responsive Grid` => `Responsive Grid`, etc.
- Doesn't give me a lot of confidence in your list when the first item in `Tools` has been deprecated for years. Please go through all your entries and ensure there are only maintained & awesome entries.
- `More` => `More…`
- In contributing.md:
  - `* [link-name](http://example.com/) - description` => `- [Link Name](https://example.com) - Description.`
  - `english` => `English`

**Comment by LucasBassetti** [comment]
Updated. Sorry for the mistakes and thanks for the great review.

**Comment by LucasBassetti** [comment]
Is everything ok now?

**Pull request merged**